client/src/components/history/Weekly.tsx

```diff
@@ -34,14 +34,17 @@ interface IUser {
 }
 
 const Weekly = () => {
-  const [userInfo] = useState<IUser>({
-    gender: EGender.female,
-    weight: 50,
-    height: 1.7,
-    caloriesGoal: 500,
-    waterGoal: 8,
-    age: 27,
-  });
+  const tempUserInfo = JSON.parse(localStorage.getItem("userInfo") || "");
+  const [userInfo] = useState<IUser>(
+    tempUserInfo || {
+      gender: EGender.female,
+      weight: 50,
+      height: 1.7,
+      caloriesGoal: 500,
+      waterGoal: 8,
+      age: 27,
+    }
+  );
```

Comment:
This needs to be pulled from the database from KM. not hard coded. For this is it possible to do it prior to saturday?

Reply:
okay

Reply:
ohhh I did not realized it's being saved to the local storage..... we should be loading it from the database not local storage as it won't be there every time if they clear the browser cache. it'll only exist during the sign-up session.  KM is working on putting that data into the APP state as a variable for us to use.